### PR TITLE
Rename some ng-ui-router-* packages

### PR DESCRIPTION
**Pull request by antleblanc**
# Rename some ng-ui-router-* packages

Replace both package and module name for `ng-ui-router-breadcrumb`, `ng-ui-router-layout` and `ng-ui-router-line-progress`

### :nail_care: Refactor

a2455b4 - refactor(components.ng-ui-router-layout): renamed package name
5131f54 - refactor(components.ng-ui-router-breadcrumb): renamed package name
fadaaf2 - refactor(components.ng-ui-router-line-progress): renamed package name

### :house: Internal

- No QC required.

#### ng-ui-router-layout
- package is now named as `@ovh-ux/ng-ui-router-layout`

#### ng-ui-router-breadcrumb

- package and module are now respectively named as `@ovh-ux/ng-ui-router-breadcrumb` and `ngUiRouterBreadcrumb`

#### ng-ui-router-line-progress

- package and module are now respectively named as `@ovh-ux/ng-ui-router-line-progress` and `ngUiRouterLineProgress`

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1599,6 +1599,24 @@
     rollup-pluginutils "^2.3.3"
     slash "^2.0.0"
 
+"@ovh-ux/manager-pci@^0.6.0-alpha.0":
```
Comment (antleblanc) [inline yarn.lock]: `yarn install` from `638e7cf552013681804993326826d993f320a275` causes this following change.

**Comment by antleblanc** [comment]
@jleveugle Please pay attention to the commit message. You will see that it is explicitly describe as a breaking change.

Hope it helps you!

File: yarn.lock (hunk 1 above)
Comment (FredericEspiau) [reply yarn.lock]: It should have been merged with `638e7cf552013681804993326826d993f320a275` then 😞

Comment (antleblanc) [reply yarn.lock]: Rebased on `9723d478fdb549390a75074d8047b6dac668d1a5`.

**Comment by cbourgois** [comment]
I think `ngUiRouterLayout` is the correct naming and we should rename the 3 others package.

**Comment by antleblanc** [comment]
According to your comment here is the expected output then:

**Before**:
- Folders are named: `ng-uirouter-*`
- Packages are named (package.json): `@ovh-ux/ng-uirouter-*`
- Modules are named: `ngUirouter*`

**After**:
- Folders will be renamed to: `ng-ui-router-*`
- Packages will be renamed to: `@ovh-ux/ng-ui-router-*`
- Modules will be renamed to: `ngUiRouter*`

Could you please confirm if you agree?

**Comment by antleblanc** [comment]
ping @cbourgois

**Comment by jleveugle** [comment]
Hello @antleblanc,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by antleblanc** [comment]
Since the addition of the following commit 627ee2c to the `develop` branch, I had to do a rebase on top of `develop` branch.

**Comment by antleblanc** [comment]
Since the addition of the following commit 5786ca4 to the `develop` branch, I had to do a rebase on top of `develop` branch.

**Pull request merged**